Server/DeviceManagement/DeviceManagement.Domain/UserAggregate/IUserRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace ModularHouse.Server.DeviceManagement.Domain.UserAggregate;
+
+public interface IUserRepository
+{
+    public DbSet<User> Users { get; }
+    public DbContext Context { get; }
```

Comment:
Add methods `Create`, `Update` and `Delete` instead of this behaviour

Reply:
Additionally, you can add method `SaveChangesAsync()`, because Repository Interface must not know anything about DbContext